### DISCUSSION TITLE
Upload to Hugging Face Hub?

### DISCUSSION THREAD

**Discussion by nalzok** [p-lambda/wilds#126] [Ideas]
Are you interested in uploading the datasets to the [Hugging Face Hub](https://huggingface.co/datasets)? That would make WILDS even more accessible.

**Comment by kohpangwei** [comment]
Yes, we'd be interested! Our sense was that it would take a fair amount of work, though. Do you have a sense of what it might involve?

**Reply by nalzok** [reply]
I'm not sure either. My understanding is that we need to provide a [dataset loading script](https://huggingface.co/docs/datasets/dataset_script) or something.

> Write a dataset script to load and share your own datasets. It is a Python file that defines the different configurations and splits of your dataset, as well as how to download and process the data.
